op-guide/configuration.md

```diff
@@ -5,232 +5,323 @@ category: operations
 
 # Configuration Flags
 
-TiDB/TiKV/PD are configurable through command-line flags and environment variables.
-
+TiDB, TiKV and PD are configurable using command-line flags and environment variables.
 
 ## TiDB
 
 The default TiDB ports are 4000 for client requests and 10080 for status report.
 
-### `--store`
-+ the storage engine type
-+ Human-readable name for this member.
-+ default: "goleveldb"
-+ You can choose from "memory", "goleveldb", "BoltDB" or "TiKV". The first three are all local storage engines. TiKV is a distributed storage engine.
+### `--binlog-socket`
 
-### `--path`
-+ the path to the data directory for local storage engines like goleveldb, BoltDB, or memory or the DSN for the distributed storage engine like TiKV. If you use TiKV, specify the path in the following format: $Host:$Port.
-+ default: "/tmp/tidb"
+- The TiDB services use the unix socket file for internal connections, such as the PUMP service
+- Default: ""
+- You can use "/tmp/pump.sock" to accept the communication of PUMP unix socket file.
+
+### `--cross-join`
+
+- To enable(true) or disable(false) the cross join without any equal condition
+- Default: true
+- The value can be (true) or (false). (true) is to enable join without any equal conditions. (false) is to disable it.
+
+### `--host`
+
+- The host address that the TiDB server monitors
+- Default: "0.0.0.0"
+- The TiDB server monitors this address.
+- The "0.0.0.0" monitors all network cards addresses. If you have multiple network cards, specify the network card that provides service, such as 192.168.100.113.
```

Comment:
The "0.0.0.0" monitors all network cards addresses -> The "0.0.0.0" monitors all network cards.